javascripts/discourse/components/post-wizard.js

```diff
@@ -2,17 +2,17 @@
 // import { ajax } from "discourse/lib/ajax";
 // import { popupAjaxError } from "discourse/lib/ajax-error";
 import Component from "@ember/component";
+import { tagName } from "@ember-decorators/component";
 import SiteSetting from "admin/models/site-setting";
 
-export default Component.extend({
-  tagName: "",
-  layoutName: "components/post-wizard",
```

Comment:
Potential breakage? (if something extends this it will be missing a template)

Reply:
This was a manual change - not a codemod thing.

I moved the template to be colocated, so now it's associated with the class properly, and inheritance will work in the modern way (without the need for layoutName)